arbitrator/prover/src/host.rs

```diff
@@ -81,6 +82,7 @@ impl FromStr for Hostio {
             ("env", "wavm_read_delayed_inbox_message") => WavmReadDelayedInboxMessage,
             ("env", "wavm_read_hotshot_commitment") => WavmReadHotShotCommitment,
             ("env", "wavm_halt_and_set_finished") => WavmHaltAndSetFinished,
+            ("wasi_snapshot_preview1", "sched_yield") => MockSchedYield,
```

Comment:
`hostio` is a place for the prover machine to get the external information. These function signatures are converted to the OpCodes directly. It seems that this function is just to make the compiler happy, so I think a better place for mocking this function is [here](https://github.com/EspressoSystems/nitro-espresso-integration/blob/a64afd84f635df27eb84e040c5eb2fe3dab92f97/arbitrator/wasm-libraries/wasi-stub/src/lib.rs#L21)

Reply:
I'm getting:
```
Error: no such hostio sched_yield in wasi_snapshot_preview1
```
So I believe we do need to mock the function here as well, unless there is a better way to make the compiler happy.

Reply:
I think you need to rebuild the wasi_stub.wasm
```
rm target/machines/latest/wasi_stub.wasm
make target/machines/latest/wasi_stub.wasm
```

Not sure why this stuff can not be rebuilt by `make build-replay-env`

Reply:
nice that worked!